examples/fedpca_examples/perform_pca/config.yaml

```diff
@@ -3,10 +3,11 @@ n_server_rounds: 1 # The number of rounds to run FL
 
 # Parameters that describe clients
 n_clients: 4 # The number of clients in the FL experiment
-batch_size: 32 # The batch size for client training
+batch_size: 32 # The batch size used for dataloader creation. (This parameter is otherwise not used in the execution of federated PCA.)
 
-low_rank: False
-full_svd: False
-rank_estimation: 6
-center_data: True
-num_components_eval: 16
+# The following are relevant technical parameters related to performing PCA.
+low_rank: False # Indicates whether the user wish to assume the input data has low rank.
+full_svd: False # Indicates whether to perform full SVD or reduced SVD.
+rank_estimation: 6 # Indicates the estimated rank of the data matrix. Only used if low_rank is True.
+center_data: True #  Indicates whether the PCA module should center the data before performing PCA.
+num_components_eval: 16 # Indicates the number of principal components used during evaluation when computing reconstruction error.
```

Comment:
Perhaps state that if it isn't in the config it defaults to none and all components are used?